test-files/cmd/slow/serial/ServiceCalculateTilesCmdTest.sh

```diff
@@ -21,3 +21,4 @@ diff $GOLD_FILES_DIR/output-cropped.geojson $OUTPUT_DIR/output-cropped.geojson
 
 # Clean up the map from the database
 hoot delete-map -D api.db.email=test@test.com $HOOT_DB_URL"/DcTigerRoads"
+scripts/database/CleanAndInitializeOsmApiDb.sh
```

Comment:
Cleaning up the OSM API database after the test.  Since there is only one OSM API database all loads and accesses must be done serially so a cleanup at the end is ok.